app/src/main/res/raw/disclaimer.txt

```diff
@@ -0,0 +1 @@
+The Adjustable Audio application was developed by a software development team. The application is a tool which conducts amateur hearing tests and should not be used as a medical standard in any situation. If the user is concerned about potential hearing loss, he or she should contact a professional audiologist or seek emergency medical attention if necessary. The application also provides methods to export sensitive medical data to different applications by request of the user. The user should only do so if he or she is comfortable sharing their personal medical data.
```

Comment:
Thought we should be a little more descriptive in some areas:

This software is provided on as-is basis without any warranty, responsibility, or liability. While the application is a tool which conducts amateur hearing tests, it should not be used as a medical standard in any situation. Moreover, the software is not, in any circumstance, intended for use in the diagnosis of disease or other conditions, or in the cure, mitigation, treatment, or prevention of disease. If the user is concerned about potential hearing loss, he or she should contact a professional audiologist or seek emergency medical attention if necessary. This software provides methods to export sensitive medical data to different applications by request of the user. The user should only do so if he or she is comfortable sharing their personal medical data.